src/atomist.config.ts

```diff
@@ -48,6 +49,10 @@ export const configuration: Configuration = {
     teamIds: extractWorkspacesFromEnvironment(),
     commands: assembled.commandHandlers.concat([]),
     events: assembled.eventHandlers.concat([]),
+    // TODO CD move ingesters to different global automation
+    ingesters: [
```

Comment:
Will this prevent people from running it? 
If so
I think we have to move it before merging

Reply:
It won’t for as long as we don’t deploy global and durable. I was planning to deal with this separately.

Reply:
This seems OK to me